### PR TITLE
Updated CharacterReset + InfiniteChestsV3 + Ghost + NPCBlocker + AutoRegister

### PR DESCRIPTION
**Pull request by moisterrific**
due to popular request...i think?

decided to update infinite chests myself bc it's such an easy change to make it work

### CODE REVIEW

**Comment by hakusaro** [comment]
@moisterrific will check soon (sorry for delay, etc., update, security, etc)

**Comment by moisterrific** [comment]
> @moisterrific will check soon (sorry for delay, etc., update, security, etc)

it's ok, no rush

**Review by hakusaro — CHANGES_REQUESTED** [review]
AutoRegister has been rejected and has failed secure code review. While not introduced by you, it needs to be updated to use a secure method of password generation. The current method is not secure. If it needs to generate passwords, it needs to use a secure PRNG instead of a password formed of composite elements. Currently, the password can easily be obtained by a malicious server.

File: README.md [hunk 1]
```diff
@@ -147,12 +147,38 @@ This is the TShock plugin repository. It collects links to all plugins in the ec
...
+#### Updated for current TShock version (4.5.3.0 Blood Moon edition)
 * [AutoRegister](https://github.com/moisterrific/TShockAutoRegister) by [brian91292](https://github.com/brian91292), updated by [moisterrific](https://github.com/moisterrific)
-  * Automatically register new users on SSC enabled servers.
-  * Built on TShock 4.4.0 Pre-11.
-  * [Download](https://files.catbox.moe/819j8p.zip)
+  * Automatically register new users on servers requiring login.
+  * Built and tested on TShock 4.5.3.0.
+  * [Download](https://files.catbox.moe/jlt9c5.zip)
   * [Source code](https://github.com/moisterrific/TShockAutoRegister)
+* [NPC Blocker](https://github.com/moisterrific/NPCBlocker2) by [Olink](https://github.com/Olink), updated by [moisterrific](https://github.com/moisterrific)
```
Comment (hakusaro) [inline README.md:157]: NPC Blocker is good

File: README.md [hunk 2]
```diff
@@ -147,12 +147,38 @@ This is the TShock plugin repository. It collects links to all plugins in the ec
...
+  * Allows you to block certain NPCs from spawning in the world.
+  * Built and tested on TShock 4.5.3.0
+  * [Download Version 2.0.0](https://files.catbox.moe/tfy6tb.zip)
+  * [Documentation](https://github.com/moisterrific/NPCBlocker2/blob/main/README.md)
+  * [Source Code](https://github.com/moisterrific/NPCBlocker2/blob/main/NPCBlocker/NPCBlocker.cs)
+* [Character Reset (SSC)](https://github.com/bippity/CharacterReset) by [Bippity](https://github.com/bippity), updated by [moisterrific](https://github.com/moisterrific)
```
Comment (hakusaro) [inline README.md:163]: Character reset is good (seefnu)

File: README.md [hunk 3]
```diff
@@ -147,12 +147,38 @@ This is the TShock plugin repository. It collects links to all plugins in the ec
...
+  * Allows you to reset SSC player data without the hassle of database editing. 
+  * Built and tested on TShock 4.5.2.0 (April Lyrids edition).
+  * [Download Version 1.0.3](https://files.catbox.moe/seefnu.zip)
+  * [Documentation](https://github.com/moisterrific/CharacterReset/blob/master/README.md)
+  * [Source Code](https://github.com/moisterrific/CharacterReset/blob/master/CharacterReset.cs)
+* [Ghost](https://github.com/DannyDan77/Ghost) by [DannyDan77](https://github.com/DannyDan77), updated by [moisterrific](https://github.com/moisterrific)
```
Comment (hakusaro) [inline README.md:169]: Ghost is good (rh3myj)

File: README.md [hunk 4]
```diff
@@ -147,12 +147,38 @@ This is the TShock plugin repository. It collects links to all plugins in the ec
...
+  * Allows admins to become invisible to other players.
+  * Built and tested on TShock 4.4.0 Pre-11.
+  * [Download Version 1.2.0](https://files.catbox.moe/rh3myj.zip)
+  * [Documentation (pls read!)](https://github.com/moisterrific/Ghost/blob/master/README.md)
+  * [Source Code](https://github.com/moisterrific/Ghost/blob/master/GhostPlugin/GhostPlugin.cs)
+* [InfiniteChestsV3](https://github.com/Zaicon/InfiniteChestsV3) by [Zaicon](https://github.com/Zaicon), updated by [moisterrific](https://github.com/moisterrific)
```
Comment (hakusaro) [inline README.md:175]: InfiniteChests is good (5zjurx)

**Comment by hakusaro** [comment]
Everything here is good except for AutoRegister. You can update that one if you want, or we can remove it from this PR and just merge the other plugins.

**Comment by moisterrific** [comment]
pls just merge the others for now, I'll fix AutoRegister later

**Review by hakusaro — APPROVED** [review]

**Pull request merged**